doc/Example of Cross Compiling the Azure IoT SDK for Python.md

```diff
@@ -0,0 +1,230 @@
+# Cross Compiling Azure IoT SDK for Python Example
+## Overview
+
+The goal of this document is to describe the steps required to cross compile the Azure IoT SDK for Python in order to allow the user to use Python to send and receive messages between the device and one&#39;s Azure IoT Hub. For this example, the target device will be the [MultiTech device](https://catalog.azureiotsuite.com/details?title=MultiConnect-Conduit&amp;source=home-page). It will describe how to install the toolchain for the this device, build the required prerequisite software and finally cross compile the Azure IoT SDK for Python library. The procedure for other Linux based target devices should be very similar.
+
+This process will consist of three tasks:
+
+1. Download and install the MultiTech toolchain
+2. Download and build the Boost library against the MultiTech toolchain
+3. Download and build the Azure IoT SDK for Python against the MultiTech toolchain
+
+All commands will assume that you are starting with your current directory at your home directory. If you keep the directories names and relative locations the same as described this will save you from editing files later in the process. To switch to your home directory enter:
+```
+cd ~
+```
+Please note that, due to width restrictions of this document, some of the entered commands may wrap on the page. They should be entered as a single line on the console.
```

Comment:
>due to width restrictions [](start = 18, length = 25)

"due to line length limits"

Reply:
Actually they don't wrap in markdown so I've deleted the line.

---
In reply to: [142532409](https://github.com/Azure/azure-iot-sdk-python/pull/76#discussion_r142532409) [](ancestors = 142532409)